homeassistant/components/yamaha_musiccast/translations/select.de.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Translations should be introduced via lokalise: https://developers.home-assistant.io/docs/translations